scripts/translation_postprocessing.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+
+"""Translation Post-processing
+
+(c) 2022 Okta, Inc.
+Author: Paul Wallace (paul.wallace@okta.com)
+
+Returned translations of HTML content require post-processing prior to
+deployment. There are a number of string segments that are not currently
+picked up in XTM, and thus are returned untranslated. 
+These segments include:
+
+* Attribute values (such as aria-labels)
+* URL fragements for locale-specific paths (such as `/en-us/`
+* URL query parameters for language (such as `?language=en_US`) 
+
+This script can be used to execute a search-and-replace of these segments. 
+
+The segments that require fixing are maintained in an accompanying
+JSON file (`en2ja.json`). String replacement pairs can be revised or added 
+therein. If in future we need to support additional languages, we can 
+add similar files for those, which can be sent to translation
+if necessary.
```

Comment:
instead of adding similar files, we could (once we remove the aria label translations) pass a parameter(s) to your  script with the language code, and based on that we do all the replacements we need

Reply:
Yes, that would be better (I'm tired of entering the JSON file each time, I'd rather just enter a directory path and have the script "do the right/expected thing".)
I can make this change now, or we could create an enhancement ticket and make this change later. Preference?

Reply:
+1 for making this enhancement later. Current approach is simple and can be improved easily when we get french translation.

Reply:
Glad we all agree. Can you update the comment please? Instead of 
```
If in future we need to support additional languages, we can 
add similar files for those, which can be sent to translation
if necessary.
```
something like:
```
If in future we need to support additional languages, we can 
change the script to pass the language as an argument
```